assets/js/state/sagas/softwareUpdates.test.js

```diff
@@ -0,0 +1,88 @@
+import { faker } from '@faker-js/faker';
+
+import { recordSaga } from '@lib/test-utils';
+
+import { networkClient } from '@lib/network';
+import MockAdapter from 'axios-mock-adapter';
+
+import {
+  startLoadingSoftwareUpdates,
+  setSoftwareUpdates,
+  setSoftwareUpdatesErrors,
+  setEmptySoftwareUpdates,
+} from '@state/softwareUpdates';
+
+import { fetchSoftwareUpdates } from './softwareUpdates';
+
+describe('Software Updates saga', () => {
+  describe('Fetching Software Updates', () => {
+    it('should successfully fetch software updates', async () => {
+      const axiosMock = new MockAdapter(networkClient);
+      const hostId = faker.string.uuid();
+      const successfulResponse = [
```

Comment:
just to keep track: a successful response would have relevant patches and upgradable packages splitted like:
```js
{
  relevant_patches: [],
  upgradable_packages: [],
}
```